lib/placebo.ex

```diff
@@ -121,24 +108,35 @@ defmodule Placebo do
       import Placebo.Matchers
       import Placebo.Validators
       import Placebo.Actions
+      require Placebo.Macros
 
-      setup do
-        on_exit(fn ->
-          mocks = Placebo.Server.get()
+      setup(context) do
+        async? = Map.get(context, :async, false)
 
+        if async? == false do
```

Comment:
This is nitpicky, but can we change to `unless async?` or `if not async?`?

Reply:
I'm afraid this nitpicky comment is going to spawn an `IfServer` implementation for placebo, but I'm going to ask for this change anyway.